### ISSUE TITLE
Changing slider values is not announced in NVDA/Win

### ISSUE THREAD

**Issue by patrickhlauke** [ThePacielloGroup/CCAe#37]
While using JAWS the new value is announced nicely, this is not happening in NVDA. Likely a bug/shortcoming in NVDA, but perhaps we can help it along with a live region around the matching text entry box next to it?

**Comment by patrickhlauke** [comment]
Seems resolved in current NVDA 2020.1
